packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -2971,9 +2971,9 @@ describe('lib/senders/emails:', () => {
   describe('mock failing sendMail method:', () => {
     beforeEach(() => {
       mailer.localize = () => ({});
-      sinon
-        .stub(mailer.mailer, 'sendMail')
-        .callsFake((config, cb) => cb(new Error('Fail')));
+      sinon.stub(mailer.mailer, 'sendMail').callsFake((config, cb: any) => {
```

Comment:
should this be `_config` to match 2944 above?
(assuming the underscore was to denote it's unused, which it might now be)